src/main/java/org/jitsi/jicofo/bridge/JvbDoctor.java

```diff
@@ -129,6 +130,11 @@ public void bridgeAdded(Bridge bridge)
         logger.info("Scheduled health-check task for: " + bridge);
     }
 
+    @Override
+    public void bridgeIsShuttingDown(@NotNull Bridge bridge)
```

Comment:
Why is this needed? Does the default Kotlin implementation not apply to Java implementations of interfaces?

Reply:
The kotlin function needs a `@JvmDefault` annotation, but that requires to run with `-Xjvm-default` and I didn't want to do that.